tests/constants/vm/vm.go

```diff
@@ -0,0 +1,23 @@
+// Copyright 2017 VMware, Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// A home to hold misc test constants related to VM (docker host)
+
+package vm
```

Comment:
this should be added to https://github.com/vmware/docker-volume-vsphere/blob/master/vmdk_plugin/Makefile#L161